nvtabular/inference/graph/ops/workflow.py

```diff
@@ -0,0 +1,101 @@
+#
+# Copyright (c) 2021, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import pathlib
+import warnings
+
+from nvtabular import Workflow
+from nvtabular.graph.schema import Schema
+from nvtabular.graph.selector import ColumnSelector
+from nvtabular.inference.graph.ops.operator import InferenceOperator
+from nvtabular.inference.triton.ensemble import (
+    _generate_column_types,
+    _generate_nvtabular_config,
+    _remove_columns,
+    _triton_datatype_to_dtype,
+)
+
+
+class WorkflowOp(InferenceOperator):
+    def __init__(
+        self, workflow, name=None, sparse_max=None, max_batch_size=None, label_columns=None
+    ):
+        self.workflow = workflow
+        self.sparse_max = sparse_max or {}
+        self.name = name or self.__class__.__name__
+        self.max_batch_size = max_batch_size
+        self.label_columns = label_columns or []
+
+    @classmethod
```

Comment:
Same issue with `transform` and `from_config` here: the code is correct as written, but we can probably do without it if we use the class hierarchy a bit more cleverly.